### PR TITLE
WIP

### PR DESCRIPTION
**Pull request by LiorAgnin**
## Types of changes

What types of changes does your code introduce?
_Put an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] Refactor or a modification (non-breaking change)
- [ ] New feature (non-breaking change which adds functionality)

### CODE REVIEW

File: dapp/src/components/FuseDashboard/components/RewardUserForm/index.jsx [hunk 1]
```diff
@@ -18,6 +23,12 @@ const RewardUserForm = ({ initialValues, setJoinBonus, setBackupBonus, setInvite
     }
   }
 
+  const initialValues = React.useMemo(() => ({
+    joinBonus: { ...get(plugins, 'joinBonus.joinInfo'), isActive: get(plugins, 'joinBonus.isActive') },
```
Comment (leonprou) [inline dapp/src/components/FuseDashboard/components/RewardUserForm/index.jsx:27]: you don't have mobx warnings here?
I learned it's problematic to use  observables `React.useMemo`
IMO we can just skip the useMemo here

File: dapp/src/components/FuseDashboard/modals/ImportExistingEntity/index.jsx [hunk 2]
```diff
@@ -44,7 +44,7 @@ const ImportExistingEntity = ({ entity, submitEntity }) => {
       validationSchema={Scheme}
       render={renderForm}
       onSubmit={onSubmit}
-      isInitialValid={false}
+      validateOnMount
```
Comment (leonprou) [inline dapp/src/components/FuseDashboard/modals/ImportExistingEntity/index.jsx:47]: nice, that's for Formik warnings?

**Review by leonprou — COMMENTED** [review]
Looks good! 
see my comment about warnings, might need small change

File: dapp/src/components/FuseDashboard/components/RewardUserForm/index.jsx (hunk 1 above)
Comment (LiorAgnin) [reply dapp/src/components/FuseDashboard/components/RewardUserForm/index.jsx:27]: Yes, you're right.
good catch, thanks.

File: dapp/src/components/FuseDashboard/modals/ImportExistingEntity/index.jsx (hunk 2 above)
Comment (LiorAgnin) [reply dapp/src/components/FuseDashboard/modals/ImportExistingEntity/index.jsx:47]: Yes

**Pull request merged**